zstacklib/zstacklib/scripts/rbd_rw.py

```
@@ -0,0 +1,242 @@
+        time.sleep(0.05)
+    if not completion.is_complete():
+        raise Exception("read image timeout after 5 seconds")
+    completion.wait_for_complete_and_cb()
+    if completion.get_return_value() != size:
+        raise Exception("read image not completed, size: {} return code: {}".format(size, completion.get_return_value()))
+    return data_ptr.raw
+
+
+def _do_write_image(image, offset, content):
+    completion = image.aio_write(content, offset, lambda _: None)
+    for i in xrange(100):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)